### ISSUE TITLE
Unable to add emails and affiliations to authors in title of paper.

### ISSUE THREAD

**Issue by purukaushik** [Hoverbear/acm-pandoc-paper#2]
Hi @Hoverbear , love this repo. I need some help in adding email addresses and university affiliations in the markdown title. I added them up like this 
```markdown
---
title: Distributed and Parallel Databases Project Report
author:
 - name: Purushotham Kaushik Swaminathan
   email: purukaushik@asu.edu
 tags: [Distributed Databases]
abstract: 
```
and it didn't work

**Comment by Hoverbear** [comment]
Ah, yes, unfortunately I never added functionality to actually use `author` as a `name` and `email` set. :( If you look [here](https://raw.githubusercontent.com/Hoverbear/acm-pandoc-paper/master/paper.md) you can see it's just 

```yml
author:
  - Andrew Hobden
```

You can refer to [this link](http://pandoc.org/MANUAL.html#extension-yaml_metadata_block) about the metadata block.

As noted at the end of that section some changes are needed to the template to support multiple fields as you'd like. [This](https://github.com/Hoverbear/acm-pandoc-paper/blob/master/templates/sig-alternate.cls) is *probably* the file which would need changing.

Unfortunately it's been quite some time for me since I touched this and LaTeX templates such as that are hardly friendly to work with. How are you with them?

**Comment by purukaushik** [comment]
Sure Andrew, will look at the metadata stuff.

Yeah, I find LaTeX templates unfriendly to work with,. I am a markdown guy. But I would want to fix this to get my final report done. If I get something to work, will submit  a PR.

**Comment by Hoverbear** [comment]
Awesome. :) Yeah, please let me know if I can help or you'd like to collaborate on it. COuld also pick a different template which supports this out of the box. Should be easy to do, just change the makefile. Hardest part would be finding the template.

**Comment by purukaushik** [comment]
@Hoverbear  -- I was able to fix (#3)  the mutliple author, affiliation, emails issue with a modification on the default.latex template of pandoc and a metadata.yaml file. 
It works for me now:
<img width="783" alt="screen shot 2016-12-18 at 2 08 49 am" src="https://cloud.githubusercontent.com/assets/5105099/21292556/06edcb80-c4c7-11e6-9507-efe6cf12e713.png">
